rocAL/rocAL/source/meta_data/meta_node_ssd_random_crop.cpp

```diff
@@ -50,12 +50,9 @@ void SSDRandomCropMetaNode::update_parameters(pMetaDataBatch input_meta_data)
     vxCopyArrayRange((vx_array)_y1, 0, _batch_size, sizeof(uint),_y1_val.data(), VX_READ_ONLY, VX_MEMORY_TYPE_HOST);
     for(int i = 0; i < _batch_size; i++)
     {
-        auto bb_count = input_meta_data->get_bb_labels_batch()[i].size();
-        int labels_buf[bb_count];
-        BoundingBoxCords box_coords_buf;
-        box_coords_buf.resize(bb_count);
-        memcpy(labels_buf, input_meta_data->get_bb_labels_batch()[i].data(),  sizeof(int)*bb_count);
-        memcpy((void *)box_coords_buf.data(), input_meta_data->get_bb_cords_batch()[i].data(), input_meta_data->get_bb_cords_batch()[i].size() * sizeof(BoundingBoxCord));
+        auto bb_count = input_meta_data->get_labels_batch()[i].size();
+        Labels labels_buf = input_meta_data->get_labels_batch()[i];
+        BoundingBoxCords box_coords_buf = input_meta_data->get_bb_cords_batch()[i];
         BoundingBoxCords bb_coords;
         Labels bb_labels;
         BoundingBoxCord crop_box;
```

Comment:
Will the below lines work without any issues for double bboxes?

Reply:
Will check and issue this in a different PR